<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Smart</class>
 <widget class="QWidget" name="Smart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
	font-family: &quot;Helvetica&quot;;
	font-size: 14pt;
	color: rgb(255, 255, 255);
	background-color: #333333;
	border: 10px;
	border-radius: 25px;
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #444444;
	border: 10px;
	border-radius: 25px;
}

QPushButton:pressed{
	color: rgb(255, 255, 255);
	background-color: #555555;
	border: 10px;
	border-radius: 25px;
}</string>
  </property>
  <widget class="QPushButton" name="pushButton_0">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>505</y>
     <width>105</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">padding-right:55px;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>450</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>450</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>450</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>395</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>395</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>395</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>340</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>340</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>340</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mult">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>340</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-family: &quot;Helvetica&quot;;
	font-size: 14pt;
	color: rgb(255, 255, 255);
	background-color: #FF9F0A;
	border: 10px;
	border-radius: 25px;
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(254, 194, 105);
	border: 10px;
	border-radius: 25px;
}

QPushButton:focus{
	color: #FF9F0A;
	background-color: rgb(255, 255, 255);
	border: 10px;
	border-radius: 25px;
}</string>
   </property>
   <property name="text">
    <string>×</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pow">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>285</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-family: &quot;Helvetica&quot;;
	font-size: 14pt;
	color: rgb(255, 255, 255);
	background-color: #212121;
	border: 10px;
	border-radius: 25px;
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #313131;
	border: 10px;
	border-radius: 25px;
}

QPushButton:pressed{
	color: rgb(255, 255, 255);
	background-color: #414141;
	border: 10px;
	border-radius: 25px;
}</string>
   </property>
   <property name="text">
    <string>x²</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ln">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-family: &quot;Helvetica&quot;;
	font-size: 14pt;
	color: rgb(255, 255, 255);
	background-color: #212121;
	border: 10px;
	border-radius: 25px;
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #313131;
	border: 10px;
	border-radius: 25px;
}

QPushButton:pressed{
	color: rgb(255, 255, 255);
	background-color: #414141;
	border: 10px;
	border-radius: 25px;
}</string>
   </property>
   <property name="text">
    <string>ln</string>
   </property>
  </widget>
  <widget class="QPushButton" name="minus">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>395</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-family: &quot;Helvetica&quot;;
	font-size: 14pt;
	color: rgb(255, 255, 255);
	background-color: #FF9F0A;
	border: 10px;
	border-radius: 25px;
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(254, 194, 105);
	border: 10px;
	border-radius: 25px;
}

QPushButton:focus{
	color: #FF9F0A;
	background-color: rgb(255, 255, 255);
	border: 10px;
	border-radius: 25px;
}</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="plus">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>450</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-family: &quot;Helvetica&quot;;
	font-size: 14pt;
	color: rgb(255, 255, 255);
	background-color: #FF9F0A;
	border: 10px;
	border-radius: 25px;
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(254, 194, 105);
	border: 10px;
	border-radius: 25px;
}

QPushButton:focus{
	color: #FF9F0A;
	background-color: rgb(255, 255, 255);
	border: 10px;
	border-radius: 25px;
}</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="result">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>505</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-family: &quot;Helvetica&quot;;
	font-size: 14pt;
	color: rgb(255, 255, 255);
	background-color: #FF9F0A;
	border: 10px;
	border-radius: 25px;
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(254, 194, 105);
	border: 10px;
	border-radius: 25px;
}

QPushButton:pressed{
	color: rgb(255,255,255);
	background-color: rgb(254, 230, 194);
	border: 20px;
	border-radius: 35px;
}</string>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="point">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>505</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sin">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>395</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-family: &quot;Helvetica&quot;;
	font-size: 14pt;
	color: rgb(255, 255, 255);
	background-color: #212121;
	border: 10px;
	border-radius: 25px;
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #313131;
	border: 10px;
	border-radius: 25px;
}

QPushButton:pressed{
	color: rgb(255, 255, 255);
	background-color: #414141;
	border: 10px;
	border-radius: 25px;
}</string>
   </property>
   <property name="text">
    <string>sin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="asin">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>395</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-family: &quot;Helvetica&quot;;
	font-size: 14pt;
	color: rgb(255, 255, 255);
	background-color: #212121;
	border: 10px;
	border-radius: 25px;
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #313131;
	border: 10px;
	border-radius: 25px;
}

QPushButton:pressed{
	color: rgb(255, 255, 255);
	background-color: #414141;
	border: 10px;
	border-radius: 25px;
}</string>
   </property>
   <property name="text">
    <string>asin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tan">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>505</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-family: &quot;Helvetica&quot;;
	font-size: 14pt;
	color: rgb(255, 255, 255);
	background-color: #212121;
	border: 10px;
	border-radius: 25px;
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #313131;
	border: 10px;
	border-radius: 25px;
}

QPushButton:pressed{
	color: rgb(255, 255, 255);
	background-color: #414141;
	border: 10px;
	border-radius: 25px;
}</string>
   </property>
   <property name="text">
    <string>tan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="atan">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>505</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-family: &quot;Helvetica&quot;;
	font-size: 14pt;
	color: rgb(255, 255, 255);
	background-color: #212121;
	border: 10px;
	border-radius: 25px;
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #313131;
	border: 10px;
	border-radius: 25px;
}

QPushButton:pressed{
	color: rgb(255, 255, 255);
	background-color: #414141;
	border: 10px;
	border-radius: 25px;
}</string>
   </property>
   <property name="text">
    <string>atan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cos">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-family: &quot;Helvetica&quot;;
	font-size: 14pt;
	color: rgb(255, 255, 255);
	background-color: #212121;
	border: 10px;
	border-radius: 25px;
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #313131;
	border: 10px;
	border-radius: 25px;
}

QPushButton:pressed{
	color: rgb(255, 255, 255);
	background-color: #414141;
	border: 10px;
	border-radius: 25px;
}</string>
   </property>
   <property name="text">
    <string>cos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="acos">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>450</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-family: &quot;Helvetica&quot;;
	font-size: 14pt;
	color: rgb(255, 255, 255);
	background-color: #212121;
	border: 10px;
	border-radius: 25px;
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #313131;
	border: 10px;
	border-radius: 25px;
}

QPushButton:pressed{
	color: rgb(255, 255, 255);
	background-color: #414141;
	border: 10px;
	border-radius: 25px;
}</string>
   </property>
   <property name="text">
    <string>acos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="log">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>340</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-family: &quot;Helvetica&quot;;
	font-size: 14pt;
	color: rgb(255, 255, 255);
	background-color: #212121;
	border: 10px;
	border-radius: 25px;
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #313131;
	border: 10px;
	border-radius: 25px;
}

QPushButton:pressed{
	color: rgb(255, 255, 255);
	background-color: #414141;
	border: 10px;
	border-radius: 25px;
}</string>
   </property>
   <property name="text">
    <string>log</string>
   </property>
  </widget>
  <widget class="QPushButton" name="c">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>285</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-family: &quot;Helvetica&quot;;
	font-size: 14pt;
	color: rgb(0, 0, 0);
	background-color: #A5A5A5;
	border: 10px;
	border-radius: 25px;
}

QPushButton:hover{
	color: rgb(0, 0, 0);
	background-color:rgb(200, 200, 200);
	border: 10px;
	border-radius: 25px;
}

QPushButton:pressed{
	color: rgb(0, 0, 0);
	background-color:rgb(230, 230, 230);
	border: 10px;
	border-radius: 25px;
}</string>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="negative">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>285</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-family: &quot;Helvetica&quot;;
	font-size: 14pt;
	color: rgb(0, 0, 0);
	background-color: #A5A5A5;
	border: 10px;
	border-radius: 25px;
}

QPushButton:hover{
	color: rgb(0, 0, 0);
	background-color:rgb(200, 200, 200);
	border: 10px;
	border-radius: 25px;
}

QPushButton:pressed{
	color: rgb(0, 0, 0);
	background-color:rgb(230, 230, 230);
	border: 10px;
	border-radius: 25px;
}</string>
   </property>
   <property name="text">
    <string>-x</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mod">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>285</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-family: &quot;Helvetica&quot;;
	font-size: 14pt;
	color: rgb(0, 0, 0);
	background-color: #A5A5A5;
	border: 10px;
	border-radius: 25px;
}

QPushButton:hover{
	color: rgb(0, 0, 0);
	background-color:rgb(200, 200, 200);
	border: 10px;
	border-radius: 25px;
}

QPushButton:pressed{
	color: rgb(0, 0, 0);
	background-color:rgb(230, 230, 230);
	border: 10px;
	border-radius: 25px;
}</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QPushButton" name="div">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>285</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-family: &quot;Helvetica&quot;;
	font-size: 14pt;
	color: rgb(255, 255, 255);
	background-color: #FF9F0A;
	border: 10px;
	border-radius: 25px;
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(254, 194, 105);
	border: 10px;
	border-radius: 25px;
}

QPushButton:focus{
	color: #FF9F0A;
	background-color: rgb(255, 255, 255);
	border: 10px;
	border-radius: 25px;
}</string>
   </property>
   <property name="text">
    <string>÷</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sqrt">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>285</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-family: &quot;Helvetica&quot;;
	font-size: 14pt;
	color: rgb(255, 255, 255);
	background-color: #212121;
	border: 10px;
	border-radius: 25px;
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #313131;
	border: 10px;
	border-radius: 25px;
}

QPushButton:pressed{
	color: rgb(255, 255, 255);
	background-color: #414141;
	border: 10px;
	border-radius: 25px;
}</string>
   </property>
   <property name="text">
    <string>√x</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ikas_back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>104</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-family: &quot;Helvetica&quot;;
	font-size: 14pt;
	color: rgb(255, 255, 255);
	background-color: #212121;
	border: 10px;
	border-radius: 25px;
	padding-right:60px;
	padding-bottom:3px;
}
</string>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ikas">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>104</width>
     <height>50</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox{
	font-family: &quot;Helvetica&quot;;
	font-size: 12pt;
	color: rgb(255, 255, 255);
	background-color: rgba(0, 0, 0, 0);
	border: 10px;
	border-radius: 25px;
	padding-right:10px;
}

QDoubleSpinBox:hover{
	color: rgb(255, 255, 255);
	background-color: rgba(49, 49, 49, 0.2);
	border: 10px;
	border-radius: 25px;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="specialValueText">
    <string/>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="maximum">
    <double>1000.000000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="xymax_back">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>230</y>
     <width>104</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-family: &quot;Helvetica&quot;;
	font-size: 14pt;
	color: rgb(255, 255, 255);
	background-color: #212121;
	border: 10px;
	border-radius: 25px;
	padding-right:40px;
	padding-bottom:4px;
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #313131;
	border: 10px;
	border-radius: 25px;
}</string>
   </property>
   <property name="text">
    <string> Oxy</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="xymax">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>230</y>
     <width>104</width>
     <height>50</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox{
	font-family: &quot;Helvetica&quot;;
	font-size: 12pt;
	color: rgb(255, 255, 255);
	background-color: rgba(0, 0, 0, 0);
	border: 10px;
	border-radius: 25px;
	padding-right:10px;
}

QDoubleSpinBox:hover{
	color: rgb(255, 255, 255);
	background-color: rgba(49, 49, 49, 0.2);
	border: 10px;
	border-radius: 25px;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="specialValueText">
    <string/>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="maximum">
    <double>1000.000000000000000</double>
   </property>
   <property name="value">
    <double>10.000000000000000</double>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>311</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{	
	text-align: right;
	font-family:&quot;Helvetica&quot;;
	font-size:30pt;
	border:none;
	color:white;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="graph">
   <property name="geometry">
    <rect>
     <x>236</x>
     <y>230</y>
     <width>104</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-family: &quot;Helvetica&quot;;
	font-size: 14pt;
	color: rgb(255, 255, 255);
	background-color: #FF9F0A;
	border: 10px;
	border-radius: 25px;
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(254, 194, 105);
	border: 10px;
	border-radius: 25px;
}

QPushButton:pressed{
	color: rgb(255,255,255);
	background-color: rgb(254, 230, 194);
	border: 10px;
	border-radius: 25px;
}</string>
   </property>
   <property name="text">
    <string>Graph</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>331</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border:none;
	
	background-color: rgb(0, 0, 0);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>ikas_back</zorder>
  <zorder>pushButton_0</zorder>
  <zorder>pushButton_1</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>pushButton_5</zorder>
  <zorder>pushButton_6</zorder>
  <zorder>pushButton_7</zorder>
  <zorder>pushButton_8</zorder>
  <zorder>pushButton_9</zorder>
  <zorder>mult</zorder>
  <zorder>pow</zorder>
  <zorder>ln</zorder>
  <zorder>minus</zorder>
  <zorder>plus</zorder>
  <zorder>result</zorder>
  <zorder>point</zorder>
  <zorder>sin</zorder>
  <zorder>asin</zorder>
  <zorder>tan</zorder>
  <zorder>atan</zorder>
  <zorder>cos</zorder>
  <zorder>acos</zorder>
  <zorder>log</zorder>
  <zorder>c</zorder>
  <zorder>negative</zorder>
  <zorder>mod</zorder>
  <zorder>div</zorder>
  <zorder>sqrt</zorder>
  <zorder>ikas</zorder>
  <zorder>xymax_back</zorder>
  <zorder>xymax</zorder>
  <zorder>lineEdit</zorder>
  <zorder>graph</zorder>
  <zorder>back</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
